### DISCUSSION TITLE
Cors Problem only with user feature and 2FA not working

### DISCUSSION THREAD

**Discussion by Aquariius** [reconmap/reconmap#109] [Q&A]
Hi really would love to try this app, but I found a few problem, I configured the CORS settings and enviromment.js, no problem for anything except when I tried to create a user or delete one. Am I doing Something wrong ? Same Error, even with the wildcard for the cors. I am using the docker compose file.

![image](https://user-images.githubusercontent.com/83990443/176345519-7a51fef6-ddb2-4466-926e-f8d6ccca501d.png)
![image](https://user-images.githubusercontent.com/83990443/176345547-4cc2e9eb-782e-48c0-9b2a-fff4022e9eb6.png)
![image](https://user-images.githubusercontent.com/83990443/176345203-d5016680-9d68-465a-a9fd-43a68e38b633.png)

Second problem, when I activate de 2FA, and tried to log back-in with the newly activated 2fa user, it do nothing, I enter the username and password, but no 2fa QR code, it bring me back to the login screen, but if I enter the admin who doesn't have the 2fa enabled, it login easily

**Comment by santiagolizardo** [comment]
Thanks for sharing the above. I am closing the discussion as the comment applies to a very old version of the app.
